crates/bdk/src/wallet/coin_selection.rs

```diff
@@ -188,7 +193,7 @@ pub trait CoinSelectionAlgorithm: core::fmt::Debug {
         optional_utxos: Vec<WeightedUtxo>,
         fee_rate: FeeRate,
         target_amount: u64,
-        drain_script: &Script,
+        drain_script: &ScriptBuf,
```

Comment:
@evanlinjin already mentioned it but just FYI the `ScriptBuf` is to `Script` as `PathBuf` is to `Path` and `String` is to `&str` - so typically one would always using either `&Script` or `ScriptBuf` depending on whether you want to own the script data or not.